nautobot_firewall_models/api/serializers.py

```diff
@@ -368,7 +327,7 @@ def validate(self, data):
         """Overload validate to pop field for custom m2m relationship."""
         # Remove custom fields data and tags (if any) prior to model validation
         attrs = data.copy()
-        attrs.pop("policyrulem2m_set", None)
+        # attrs.pop("policyrulem2m_set", None)
```

Comment:
??

Reply:
No longer needed since index is on the rule. Will remove